docs/architecture/microservices/implement-resilient-applications/implement-http-call-retries-exponential-backoff-polly.md

```diff
@@ -1,22 +1,22 @@
 ---
 title: Implement HTTP call retries with exponential backoff with Polly
-description: Learn how to handle HTTP failures with Polly and HttpClientFactory.
+description: Learn how to handle HTTP failures with Polly and IHttpClientFactory.
 ms.date: 01/30/2020
 ---
 
-# Implement HTTP call retries with exponential backoff with HttpClientFactory and Polly policies
+# Implement HTTP call retries with exponential backoff with IHttpClientFactory and Polly policies
 
 The recommended approach for retries with exponential backoff is to take advantage of more advanced .NET libraries like the open-source [Polly library](https://github.com/App-vNext/Polly).
 
 Polly is a .NET library that provides resilience and transient-fault handling capabilities. You can implement those capabilities by applying Polly policies such as Retry, Circuit Breaker, Bulkhead Isolation, Timeout, and Fallback. Polly targets .NET Framework 4.x and .NET Standard 1.0, 1.1, and 2.0 (which supports .NET Core).
 
-However, writing your own custom code to use Polly’s library with HttpClient can be significantly complex. In the original version of eShopOnContainers, there was a [ResilientHttpClient building-block](https://github.com/dotnet-architecture/eShopOnContainers/commit/0c317d56f3c8937f6823cf1b45f5683397274815#diff-e6532e623eb606a0f8568663403e3a10) based on Polly. But with the release of [HttpClientFactory](use-httpclientfactory-to-implement-resilient-http-requests.md), implementing resilient HTTP communication with Polly has become much simpler, so that building-block was deprecated from eShopOnContainers.
+However, writing your own custom code to use Polly’s library with HttpClient can be significantly complex. In the original version of eShopOnContainers, there was a [ResilientHttpClient building-block](https://github.com/dotnet-architecture/eShopOnContainers/commit/0c317d56f3c8937f6823cf1b45f5683397274815#diff-e6532e623eb606a0f8568663403e3a10) based on Polly. But with the release of [IHttpClientFactory](use-httpclientfactory-to-implement-resilient-http-requests.md), implementing resilient HTTP communication with Polly has become much simpler, so that building-block was deprecated from eShopOnContainers.
```

Comment:
We should remove this para altogether. We don't have to really mention about a historic implementation that does not add any value in this context.